package.json

```diff
@@ -20,35 +20,41 @@
   "scripts": {
     "start": "backstage-cli package start",
     "build": "backstage-cli package build",
+    "export-dynamic": "janus-cli package export-dynamic-plugin",
     "lint": "backstage-cli package lint",
     "test": "backstage-cli package test",
     "clean": "backstage-cli package clean",
     "prepack": "backstage-cli package prepack",
-    "postpack": "backstage-cli package postpack"
+    "postpack": "backstage-cli package postpack",
+    "postversion": "yarn run export-dynamic"
   },
   "dependencies": {
     "@backstage/backend-common": "^0.19.3",
+    "@backstage/backend-plugin-manager": "npm:@janus-idp/backend-plugin-manager@0.0.2-janus.5",
     "@backstage/backend-tasks": "^0.5.6",
     "@backstage/catalog-model": "^1.4.1",
     "@backstage/config": "^1.0.8",
     "@backstage/plugin-catalog-backend": "^1.12.1",
     "@types/express": "*",
     "express": "^4.17.3",
     "express-promise-router": "^4.1.0",
-    "winston": "^3.2.1",
     "node-fetch": "^2.6.7",
-    "yn": "^4.0.0",
-    "js-base64": "^3.7.3"
+    "winston": "^3.2.1",
+    "yn": "^4.0.0"
```

Comment:
Coming from the previous https://github.com/microcks/microcks-backstage-provider/pull/13